src/languageservice/parser/jsonParser07.ts

```diff
@@ -1054,6 +1062,8 @@ function validate(
         case 'date':
         case 'time':
         case 'email':
+        case 'ipv4':
+        case 'ipv6':
```

Comment:
A few lines below here where the `exec` call is made, would it be better to change to `test` ? We don't really use any of the objects from `exec`.